spec/requests/mutations/users/register_user_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+module Mutations
+  module Users
+    RSpec.describe RegisterUser, type: :request do
+      describe '.resolve' do
+        it 'creates a new user' do
+          @input = {
```

Comment:
Awesome! On the last PR you had where I was asking how the password input would be "mocked" for testing, seeing this test here answered that for me.